deployment/src/main/resources/templates/additionalModelTypeAnnotations.qute

```diff
@@ -1 +1,3 @@
-{#for annon in m.additionalModelTypeAnnotations.orEmpty}{annon}{/for}
```

Comment:
I'm not an expert in using Qute so I'm not sure if this change can be done in a different way, I'm open to suggestions

Reply:
Me enither, but Im guessing what happens if you try with 
{additionalModelTypeAnnotations.orEmpty}